src/command.ts

```diff
@@ -202,72 +202,74 @@ export class Command<C extends Context = Context> implements MiddlewareObj<C> {
    */
   public addToScope(
     scope: BotCommandGroupsScope,
-    middleware: MaybeArray<ChatTypeMiddleware<C, "group" | "supergroup">>,
+    middleware?: MaybeArray<ChatTypeMiddleware<C, "group" | "supergroup">>,
     options?: Partial<CommandOptions>,
   ): this;
   public addToScope(
     scope: BotCommandScopeAllPrivateChats,
-    middleware: MaybeArray<ChatTypeMiddleware<C, "private">>,
+    middleware?: MaybeArray<ChatTypeMiddleware<C, "private">>,
     options?: Partial<CommandOptions>,
   ): this;
   public addToScope(
     scope: BotCommandScope,
-    middleware: MaybeArray<Middleware<C>>,
+    middleware?: MaybeArray<Middleware<C>>,
     options?: Partial<CommandOptions>,
   ): this;
   public addToScope(
     scope: BotCommandScope,
-    middleware: MaybeArray<Middleware<C>>,
+    middleware?: MaybeArray<Middleware<C>>,
     options: Partial<CommandOptions> = this._options,
   ): this {
```

Comment:
If there is no middleware, the plugin have no conserns nor regime about its implementation, therefore, no need for options? 

What is the porpuse of it? Setting the Command Menu with a surface-only command that its supossedly handled somewhere else outside the plugin?

Reply:
> If there is no middleware, the plugin have no concerns nor regime about its implementation, therefore, no need for options?

Hmm... true. I guess I didn't want to complicate the overloads

> What is the purpose of a no middleware command? Setting the Command Menu with a surface-only command that its supposedly handled somewhere else outside the plugin?

Exactly. TAL at the linked message. That does make sense since managing scopes can be quite cumbersome and the plugin aims to make that easier